<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mian_Interface</class>
 <widget class="QWidget" name="Mian_Interface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mian_Interface</string>
  </property>
  <widget class="QWidget" name="Mianwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>480</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>211</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_falult">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>460</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>274</x>
      <y>0</y>
      <width>250</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>主界面</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="but_menu">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>390</y>
      <width>120</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>EPS主菜单</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but_log">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>389</y>
      <width>120</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>故障/告警</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Startup_Shutdown">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>389</y>
      <width>120</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>开/关机</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_line">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>55</y>
      <width>641</width>
      <height>331</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>180</y>
      <width>71</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>A 相</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>180</y>
      <width>71</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>C 相</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>180</y>
      <width>71</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>B 相</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>180</y>
      <width>71</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>C 相</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>180</y>
      <width>81</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>B 相</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>180</y>
      <width>81</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>A 相</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Main_Electric_Frequency_A">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>270</y>
      <width>71</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>A 相</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Main_Electric_Voltage_A">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>220</y>
      <width>71</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>A 相</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Main_Electric_Frequency_C">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>270</y>
      <width>71</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>C 相</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Main_Electric_Frequency_B">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>270</y>
      <width>71</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>B 相</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>220</y>
      <width>111</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>主电电压(V):</string>
    </property>
   </widget>
   <widget class="QLabel" name="Main_Electric_Voltage_B">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>220</y>
      <width>71</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>B 相</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Main_Electric_Voltage_C">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>220</y>
      <width>71</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>C 相</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>270</y>
      <width>111</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>主电频率(Hz):</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>220</y>
      <width>121</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>输出电压(V):</string>
    </property>
   </widget>
   <widget class="QLabel" name="Output_Voltage_A">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>220</y>
      <width>71</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>A 相</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Output_Voltage_B">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>220</y>
      <width>71</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>B 相</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Output_Voltage_C">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>220</y>
      <width>71</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>C 相</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Output_Frecquency_B">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>260</y>
      <width>71</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>B 相</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>260</y>
      <width>121</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>输出频率(Hz):</string>
    </property>
   </widget>
   <widget class="QLabel" name="Output_Frecquency_C">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>260</y>
      <width>71</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>C 相</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Output_Frecquency_A">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>260</y>
      <width>71</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>A 相</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Output_Current_A">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>300</y>
      <width>70</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>A 相</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Output_Current_B">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>300</y>
      <width>70</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>B 相</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_27">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>300</y>
      <width>121</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>输出电流(A):</string>
    </property>
   </widget>
   <widget class="QLabel" name="Output_Current_C">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>300</y>
      <width>71</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>C 相</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Output_Load_Pre_C">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>340</y>
      <width>71</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>C 相</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Output_Load_Pre_B">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>340</y>
      <width>70</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>B 相</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_31">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>340</y>
      <width>121</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>负载百分比(%):</string>
    </property>
   </widget>
   <widget class="QLabel" name="Output_Load_Pre_A">
    <property name="geometry">
     <rect>
      <x>569</x>
      <y>340</y>
      <width>51</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>A 相</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="Silent_Button">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>390</y>
      <width>120</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>消音</string>
    </property>
   </widget>
   <widget class="QLabel" name="temp">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>320</y>
      <width>71</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>320</y>
      <width>111</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>环境温度(℃)：</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>but_log</tabstop>
  <tabstop>Silent_Button</tabstop>
  <tabstop>but_menu</tabstop>
  <tabstop>Startup_Shutdown</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
